private_gpt/components/vector_store/vector_store_component.py

```diff
@@ -4,10 +4,10 @@
 from injector import inject, singleton
 from llama_index.core.indices.vector_store import VectorIndexRetriever, VectorStoreIndex
 from llama_index.core.vector_stores.types import (
+    BasePydanticVectorStore,
```

Comment:
Why switching to BasePydanticVectorStore from VectorStore?

Reply:
Llama-index decision. Without that, mypy doesn't pass.
https://github.com/run-llama/llama_index/blob/main/llama-index-core/llama_index/core/vector_stores/types.py#L316